lang/de.xml

```diff
@@ -1,20 +1,265 @@
+			<month>juli</month>
+			<month>august</month>
+			<month>september</month>
+			<month>oktober</month>
+			<month>november</month>
+			<month>dezember</month>
+		</small_ymonth>
+		<hidden_trophy>Geheime Trophäe</hidden_trophy>
+		<one_hour_ago>Vor 1 Stunde</one_hour_ago>
+		<one_minute_ago>Vor 1 Minute</one_minute_ago>
+		<bronze>Bronze</bronze>
+		<gold>Gold</gold>
+		<platinum>Platin</platinum>
+		<silver>Silber</silver>
+		<hours_ago>Vor {} Stunden</hours_ago>
+		<minutes_ago>Vor {} Minuten</minutes_ago>
+	</common>
+
+	<dialog>
+		<trophy>
+			<preparing_start_app>Bereite das Starten der Anwendung vor...</preparing_start_app>
+		</trophy>
+		<save_data>
+			<delete>
+				<cancel_deleting>Möchtest du das Löschen abbrechen?</cancel_deleting>
+				<deletion_complete>Fertig gelöscht.</deletion_complete>
+				<delete_saved_data>Diese Nutzerdaten löschen?</delete_saved_data>
+			</delete>
+			<info>
+				<details>Details</details>
+				<updated>Aktualisiert</updated>
+			</info>
+			<load name="Laden">
+				<cancel_loading>Möchtest du das Laden abbrechen?</cancel_loading>
+				<no_saved_data>Keine Nutzerdaten vorhanden.</no_saved_data>
+				<load_complete>Fertig geladen.</load_complete>
+				<loading>Lade...</loading>
+				<load_saved_data>Diese Nutzerdaten laden?</load_saved_data>
+			</load>
+			<save name="Speichern">
+				<cancel_saving>Möchtest du das Speichern abbrechen?</cancel_saving>
+				<could_not_save>Konnte die Datei nicht speichern.
+Es fehlt freier Platz auf der Speicherkarte.</could_not_save>
+				<not_free_space>Es fehlt freier Platz auf der Speicherkarte.</not_free_space>
+				<new_saved_data>Neue Nutzerdaten</new_saved_data>
+				<saving_complete>Fertig gespeichert.</saving_complete>
+				<save_the_data>Diese Nutzerdaten speichern?</save_the_data>
+				<saving>Speichere...</saving>
+				<warning_saving>Speichere...
+Schalte die Konsole nicht aus und beende nicht die Anwendung.</warning_saving>
+				<overwrite_saved_data>Vorhandene Nutzerdaten überschreiben?</overwrite_saved_data>
+			</save>
+		</save_data>
+	</dialog>
+
 	<game_data>
 		<app_close>Folgende Anwendung wird geschlossen.</app_close>
 	</game_data>
 
+	<indicator>
+		<app_added_home>Die Anwendung wurde dem Startbildschirm hinzugefügt.</app_added_home>
+		<delete_all>Lösche alle</delete_all>
+		<notif_deleted>Alle Benachrichtigungen werden gelöscht.</notif_deleted>
+		<install_failed>Installieren hat nicht geklappt.</install_failed>
+		<install_complete>Fertig installiert.</install_complete>
+		<installing>Installiere...</installing>
+		<no_notif>Es gibt keine Benachrichtigungen.</no_notif>
+		<trophy_earned>Du hast eine Trophäe freigeschaltet!</trophy_earned>
+	</indicator>
+
 	<initial_setup>
 		<back>Zurück</back>
 		<completed_setup>Die Erstinstallation ist jetzt abgeschlossen.
 Dein Vita3K-System ist bereit!</completed_setup>
-		<select_language>Wählen eine Sprache.</select_language>
+		<select_language>Wähle eine Sprache.</select_language>
```

Comment:
Existing String was grammatically incorrect

Reply:
Is playstation official text from psvita....

Reply:
Sony got it wrong. Ask any other native speaker.
Btw, I didn't realize before starting that you seem to replicate the official text here.
This is mostly all my own translation.

Reply:
Yes - Sony got it wrong